newIDE/app/src/Utils/GDevelopServices/Shop.js

```diff
@@ -83,7 +83,12 @@ export const listListedPrivateGameTemplates = async ({
       withAppStoreProductId: !!onlyAppStorePrivateGameTemplates,
     },
   });
-  return response.data;
+  const gameTemplates = response.data;
+  if (!Array.isArray(gameTemplates)) {
```

Comment:
for some reason, sometimes the response is something unexpected, which is not null, and accessing a property on the object causes a crash afterwards

Reply:
Sounds good